postman/national_record_locator_producer_sandbox.json

```diff
@@ -0,0 +1,398 @@
+{
+	"info": {
+		"_postman_id": "b503cff1-8a65-48c0-8479-58900a1c6350",
+		"name": "National Record Locator Producer Sandbox",
+		"description": "Sandbox example requests",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "5426004"
+	},
+	"item": [
+		{
+			"name": "Create Pointer",
+			"protocolProfileBehavior": {
+				"disabledSystemHeaders": {
+					"accept": true
+				}
+			},
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "nhsd-correlation-id",
+						"value": "nrl-consumer-api-sandbox",
```

Comment:
change the value to 'nrl-producer-api-sandbox' throughout the file

Reply:
updated